huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go

```diff
@@ -368,53 +375,65 @@ func resourceGaussRedisInstanceV3Create(d *schema.ResourceData, meta interface{}
 
 	instance, err := instances.Create(client, createOpts).Extract()
 	if err != nil {
-		return fmtp.Errorf("Error creating GeminiDB instance : %s", err)
+		return diag.Errorf("error creating GeminiDB instance : %s", err)
+	}
+
+	var delayTime time.Duration = 120
+	// 1. wait for order success
+	if instance.OrderId != "" {
+		bssClient, err := cfg.BssV2Client(region)
+		if err != nil {
+			return diag.Errorf("error creating BSS V2 client: %s", err)
+		}
+		if err = orders.WaitForOrderSuccess(bssClient, int(d.Timeout(schema.TimeoutCreate)/time.Second),
+			instance.OrderId); err != nil {
+			return diag.FromErr(err)
+		}
+		delayTime = 10
 	}
 
 	d.SetId(instance.Id)
 	// waiting for the instance to become ready
 	stateConf := &resource.StateChangeConf{
-		Pending:      []string{"creating"},
+		Pending:      []string{"creating", "BACKUP"},
```

Comment:
can not find the pending status `BACKUP` in document.

Reply:
there may be omissions in the document, this status will be occured when the instance is backuping